spaces/swarm/skin.scss

```
@@ -0,0 +1,10 @@
+.name {
```

The css selector should similar to the space key.
```suggestion
.swarm {
```

---

fixed, sorry, I knew and still I forgot